opm/simulators/flow/Main.hpp

```diff
@@ -290,7 +288,6 @@ namespace Opm
 
             if (false) {}
 
-#ifndef FLOW_BLACKOIL_ONLY
```

Comment:
If we're really removing this check then we can also remove the `if (false) {}` check above.

Reply:
ah yes.